src/components/Menu/Menu.tsx

```diff
@@ -0,0 +1,89 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React, { PropsWithChildren, ReactElement, RefObject, useEffect, useState } from 'react';
+import { usePopper } from 'react-popper';
+import { merge } from '@utilities/merge';
+import { MenuItemProps } from '@components/MenuItem';
+import { useMenuKeyboardNavigation } from '@components/Menu/useMenuKeyboardNavigation';
+interface Props {
+    triggerRef?: RefObject<Element>;
+    open?: boolean;
+    children?: ReactElement<MenuItemProps> | ReactElement<MenuItemProps>[];
+    onClose?: () => void;
+}
+
+export type MenuProps = PropsWithChildren<Props>;
+
+const CONTAINER_BASE_CLASSES = 'tw-relative tw-bg-base tw-rounded tw-py-2 tw-shadow-mid tw-w-max';
+const CONTAINER_POSITION_CLASSES = ' tw-left-0 tw-top-7';
+const INSET_BORDER =
```

Comment:
Shouldn't this live in the theme?

Reply:
This is something I have planned but the Flyout wasn't merged already when I created this. I also want to create a helper function to add pseudo prefixes to a set of classes.
Like:
```
addPseudoCssPrefix('before', 'tw-pointer-events-none tw-block tw-rounded tw-top-0');
```
Would return
```
'before:tw-pointer-events-none before:tw-block before:tw-rounded tw-top-0'
```

Reply:
Added to the utilities. The helper function we can implement later if needed since we are using prefixes everywhere